src/Arcus.Scripting.ApiManagement/Scripts/Backup-AzApiManagementService.ps1

```diff
@@ -0,0 +1,40 @@
+param(
+    [string][parameter(Mandatory = $true)] $ResourceGroupName = $(throw "Resource group name is required"),
+    [string][parameter(Mandatory = $true)] $StorageAccountName = $(throw "Storage account name is required"),
+    [string][parameter(Mandatory = $true)] $ServiceName = $(throw "API managgement service name is required"),
+    [string][parameter(Mandatory = $true)] $TargetContainerName = $(throw "Name of the target blob container is required"),
+    [string][parameter(Mandatory = $false)] $TargetBlobName = $null,
+    [switch][parameter(Mandatory = $false)] $PassThru = $false,
+    [Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer][parameter(Mandatory = $false)] $DefaultProfile = $null
```

Comment:
As mentioned earlier, not sure if this should be included.

Reply:
I'm not sure either. 😄 Just wanted to pass along as many options as possible so the consumer has the broadest possible options from the 'origninal' Azure commands.

Reply:
Since it doesn't add any additional complexity, it can be kept. As you say, it broadens its capabilities.